mautic/mautic
# [UI] Replace page actions icons from FontAwesome to Remix

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
This PR changes icons from FontAwesome to Remix, focusing on buttons.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open all main pages and details, page dropdown, list dropdown

![image](https://github.com/mautic/mautic/assets/116097999/5a955a94-3814-43c4-b077-e31f60760190)

![image](https://github.com/mautic/mautic/assets/116097999/3a006620-bcfd-4e4a-820f-c91ca80249f5)

![image](https://github.com/mautic/mautic/assets/116097999/f456acc6-84cd-43f1-8351-8d14d8f18f96)
